site/jekyll/getting-started/tutorial.md

```diff
@@ -402,13 +402,16 @@ namespace ReactDemo.Controllers
 Let's also add a new controller action to return the list of comments:
 
 ```csharp
+[OutputCache(Location = OutputCacheLocation.None)]
 public ActionResult Comments()
 {
 	return Json(_comments, JsonRequestBehavior.AllowGet);
 }
 ```
 
-And a corresponding route in `App_Start\RouteConfig.cs`:
+The `OutputCache` attribute is used here to prevent IE from caching the ajax request. IE tries to optimize things by assuming that identical requests will return identical responses. Subsequent calls to the Comments action will simply return the cached response from the first call, the result being that the comment list is never updated in IE as new comments are added. When designing a real world API, cache times of API requests should be considered more carefully. For this tutorial it is easiest to simply disable caching.
+
+Finally we add a corresponding route in `App_Start\RouteConfig.cs` to simplify the request URL a bit:
```

Comment:
Why shouldn't a production app use the default route? I have done a couple of MVC apps and they both use default routes for almost everything. It seems to work fine.

I agree that the default route in this case looks a bit silly, but normally I think comments would be in another controller than the home controller, or maybe even a WebAPI call and that would fix the route automatically.